bibliopixel/drivers/serial_driver.py

```diff
@@ -80,7 +80,11 @@ class DriverSerial(DriverBase):
     deviceIDS = {}
     deviceVers = []
 
-    def __init__(self, type, num, dev="", c_order=ChannelOrder.RGB, SPISpeed=2, gamma=None, restart_timeout=3, deviceID=None, hardwareID="1D50:60AB"):
+    def __init__(self, type, num, dev="",
+                 c_order=ChannelOrder.RGB, SPISpeed=2,
+                 gamma=None, restart_timeout=3,
+                 deviceID=None, hardwareID="1D50:60AB",
+                 baudrate=921600):
```

Comment:
Definitely extract `921600` as a constant, because it appears three times, and because it really isn't clear where it comes from - aside from being 900 * 1024.